frontend/packages/knative-plugin/src/components/overview/__mocks__/overview-knative-mock.ts

```diff
@@ -27,3 +27,42 @@ export const mockRoutesData = [
     },
   },
 ];
+
+export const mockEventSourceData = {
+  apiVersion: 'sources.eventing.knative.dev/v1alpha1',
+  kind: 'ContainerSource',
+  metadata: {
+    name: 'test-heartbeats',
+    namespace: 'serverless-test',
+    uid: '7ad35226-1d31-4af9-8e08-7158e3469d32',
+  },
+  spec: {
+    sink: {
+      apiVersion: 'serving.knative.dev/v1alpha1',
+      kind: 'Service',
+      name: 'event-display'
+    } ,
+  },
+  status: {
+    sinkUri: 'http://event-display.serverless-test.svc.cluster.local',
+  },
+};
+
+export const mockEventSourceDataNoURI = {
+  apiVersion: 'sources.eventing.knative.dev/v1alpha1',
+  kind: 'ContainerSource',
+  metadata: {
+    name: 'test-heartbeats',
+    namespace: 'serverless-test',
+    uid: '7ad35226-1d31-4af9-8e08-7158e3469d32',
+  },
+  spec: {
+    sink: {
+      apiVersion: 'serving.knative.dev/v1alpha1',
+      kind: 'Service',
+      name: 'event-display'
+    } ,
+  },
+  status: {
+  },
+};
```

Comment:
Missing newline at end of the file and seems we need to move this in test data, take a look at review docs

`__mocks__ ` folder is for jest mocks https://jestjs.io/docs/en/manual-mocks
Keep test data in the `__tests__` dir or re-usable in the `src/test/data` dir.

Reply:
Moved the test data to _test_ folder